src/Common/src/System/Dynamic/Utils/ContractUtils.cs

```diff
@@ -13,7 +13,7 @@ public static Exception Unreachable
         {
             get
             {
-                Debug.Assert(false, "Unreachable");
+                Debug.Assert(false, nameof(Unreachable));
```

Comment:
I don't believe this is meant to be the name of the member... it's simply a comment saying that this should be unreachable.